docs/ldml/tr35-general.html

```diff
@@ -1349,29 +1349,51 @@ <h2>6 <a name="Unit_Elements" href="#Unit_Elements" id=
     west) #REQUIRED &gt;<br>
     <br>
     &lt;!ELEMENT durationUnitPattern ( #PCDATA ) &gt;<br></p>
+
     <p>These elements specify the localized way of formatting
     quantities of units such as years, months, days, hours, minutes
     and seconds— for example, in English, "1 day" or "3 days". The
     English rules that produce this example are as follows ({0}
     indicates the position of the formatted numeric value):</p>
+
     <pre>&lt;unit type="duration-day"&gt;
   &lt;displayName&gt;days&lt;/displayName&gt;
   &lt;unitPattern count="one"&gt;<span style=
 "color: blue">{0} day</span>&lt;/unitName&gt;
   &lt;unitPattern count="other"&gt;<span style=
 "color: blue">{0} days</span>&lt;/unitName&gt;
 &lt;/unit&gt;</pre>
+
     <p>In addition to supporting language-specific plural cases
     such as “one” and “other”, unitPatterns support the
     language-independent explicit cases “0” and “1” for special
     handling of numeric values that are exactly 0 or 1; see
     <a href="tr35-numbers.html#Explicit_0_1_rules">Explicit 0 and 1
     rules</a>.</p>
+
+    <p class="changed">The &lt;unitPattern&gt; elements may be used to format
+    quantities with decimal values; in such cases the choice of plural form will
+    depend not only on the numeric value, but also on its formatting (see
+    <a href="tr35-numbers.html#Language_Plural_Rules">Language Plural Rules</a>).
+    In addition to formatting units for stand-alone use,  &lt;unitPattern&gt;
+    elements are increasingly being used to format units for use in running text;
+    for such usages, the new <a href="#Grammatical_Features">Grammatical Features</a>
```

Comment:
New => developing
{We aren't far enough along yet to use}

Reply:
Thanks, made that change.